### ISSUE TITLE
RDF version of the OpenMath CDs broken when accessed via canonical URIs

### ISSUE THREAD

**Issue by kohlhase** [OpenMath/OpenMath.github.io#50]
see https://github.com/OpenMath/OM3/issues/141 
I am not sure that this is relevant any more, but we should at least follow up

**Comment by davidcarlisle** [comment]
We don't have rdf versions at all as far as I can see but there is no possibility of making different versions available at the same URI if we host at github pages. It doesn't give sufficient (or any) access to configuring handling requests for specific mime types.


Is the code under lib/krextor-xslt/  supposed to generate some rdf?
 If there is something that can be run I can add it to the travis setup to generate static .rdf versions if that is required by anyone.

**Comment by kohlhase** [comment]
yes, I think the krextor code is what was was intended here. I do not think we should try to reinstate this unless we can reactivate Christoph. I will write to him, and see what he thinks. In any case, let's not delete lib/krextor-xslt; maybe add a comment about what it is/does.
